Fix is_empty function

SongweiGe opened the pull request:
Hey I found that there is a small bug in the is_empty function that the variable creatred by torch.empty() should return t.nelement() == 1. The bug will make the uninitialized self.pl_mean never get overwritten and cause the overflow issue sometimes.

lucidrains commented:
@SongweiGe thank you! that is indeed a bug! could you change the == to a <= and i'll merge!

lucidrains commented:
@SongweiGe oh nvm, I think I just fixed the actual underlying issue

lucidrains commented:
https://github.com/lucidrains/stylegan2-pytorch/commit/611ab464d5cb40bd664712a181a7c5c6ebc5e3b3 could you try 0.17.6?

SongweiGe commented:
> @SongweiGe oh nvm, I think I just fixed the actual underlying issue

Oh I didn't realize that you have changed the `self.pl_mean = torch.empty(1).cuda()` to `self.pl_mean = 0` previously. You really did a great to keep this repo active and updated! [0fdaa3be977aad8b6c13141dfc0977e643176050](url). 

> [611ab46](https://github.com/lucidrains/stylegan2-pytorch/commit/611ab464d5cb40bd664712a181a7c5c6ebc5e3b3) could you try 0.17.6?

Thanks, this should work. It might still be good to update the is_empty function for torch Tensor anyway. : p

SongweiGe commented:
> @SongweiGe thank you! that is indeed a bug! could you change the == to a <= and i'll merge!

Oh wait, why should it be <=? If the variable is created by `torch.empty`, `t.nelement()` should return 1 and otherwise 0, right?

lucidrains commented:
Yea ignore what I said, it should be fixed now!

lucidrains commented:
I found a separate bug and fixed it lol
